tools/rewriter/SourceRewriter.cpp

```diff
@@ -1165,14 +1256,31 @@ auto ValidateDirectory = CLI::Validator(
     },
     "Directory", "path of a directory");
 
-const std::map<std::string, int> arch_map{{"x86", (int)Arch::X86}, {"aarch64", (int)Arch::Aarch64}};
+const std::map<std::string, Arch> arch_map{{"x86", Arch::X86}, {"aarch64", Arch::Aarch64}};
 
 std::string CompilationDbPath;
+std::string LibraryFilesFile;
+std::string RewriteFilesFile;
 std::vector<std::string> SourceFiles;
 std::vector<std::string> ExtraArgs;
 
+auto LibOnlyGroup = "Library-only mode";
+
 int main(int argc, const char **argv) {
   CLI::App app{"IA2 Source Rewriter"};
+  auto LibFilesOpt =
+      app.add_option("--library-files", LibraryFilesFile, "Path to file listing untrusted library sources")
```

Comment:
Since the argument is the path to a file containing the library files (and not the actual files themselves), could it have a better name like `--library-files-list-file`?

Alternatively, why not just pass in the files directly? If the command line becomes too long, there are other options: `xargs`, or we could add more generic support for `@file` that other Linux tools support.